tweets/2020/01/german-docs.tweet

```diff
@@ -0,0 +1,5 @@
+Starship's docs are now translated in German! 🇩🇪🎉
+
```

Comment:
```suggestion
```

In the Russian tweet we didn't have a new line - if you're looking for consistency

Reply:
I made this PR in the GitHub UI, which seems to auto-insert this line-break.
It looks like it doesn't show up in the preview: https://github.com/starship/twitter/pull/4/checks?check_run_id=407347834

Reply:
The link from above shows a newline for me, v.s. the preview of the Russian tweet: https://github.com/starship/twitter/pull/3/checks

Reply:
Oh, _that_ newline. I thought you meant the trailing newline. 😅 
Good call 👍

Reply:
Sorry for the miss-understanding 😅